src/server/game/Battlefield/BattlefieldMgr.cpp

```diff
@@ -40,6 +40,12 @@ BattlefieldMgr* BattlefieldMgr::instance()
 
 void BattlefieldMgr::InitBattlefield()
 {
+    if (sWorld->getIntConfig(CONFIG_WINTERGRASP_ENABLE) == -1)
```

Comment:
Hmm... tbh this is just off the top of my head to see if it will allow the workflow to past. I know negative numbers are uint, but lets see if this passes
```suggestion
    if (sWorld->getIntConfig(CONFIG_WINTERGRASP_ENABLE) < 0)
```

Reply:
I changed the value to 2 to make the workflow happy. Haha. ^_^ I do think -1 is more clear that is is "very disabled," but this will keep the compiler from complain.

Reply:
sounds good to me.